src/aurelia-binding.d.ts

```diff
@@ -509,6 +512,28 @@ export declare interface InternalCollectionObserver {
    * @param callable A callable object.
    */
   unsubscribe(context: any, callable: Callable): void;
+  /**
+   * Adds a change record to the collection observer.
+   * @param changeRecord 
+   */
+  addChangeRecord(changeRecord: ICollectionObserverSplice): void;
+  /**
+   * This will flush the change records of this observer and call any subscribers if applicable.
+   */
+  flushChangeRecords(): void;
+  /**
+   * Reset the observer to the passed collection and call any subscribers with changes between the current collection and the reset collection.
+   * @param oldCollection 
+   */
+  reset(oldCollection): void;
+  /**
+   * Get a length observer for this collection.
+   */
+  getLengthObserver(): any;
```

Comment:
This seems OK. It can make sense for a user to want the length observer, albeit very rarely.

Reply:
Okay cool! Are you good with reset being exposed?